catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/predicate/TemporalPredicate.java

```diff
@@ -102,26 +99,28 @@ public boolean matches(Event properties) {
         if (entry != null) {
             LOGGER.debug("entry id: {}", entry.getId());
 
-            switch (this.type) {
```

Comment:
Can't we still do a switch on a string? maybe a toLowerCase on it?  Seems more readable than all these if/elses